### PR TITLE
updated user-page-loader.js and user-page.html to hide messaging options on direct messages

### PR DESCRIPTION
**Pull request by kassidywebb**
Added If statements to user-page-loader.js to determine if it's a direct messaging page. If it isn't, un-hide messaging options. Also added divs to user-page.html so the user-page-loader.js knows what to hide.

### CODE REVIEW

File: src/main/webapp/js/user-page-loader.js [hunk 1]
```diff
@@ -46,6 +46,7 @@ function setPageTitle() {
 
 /**
  * Shows the message form if the user is logged in and viewing their own page.
+ * Hides private messaging option if user is on private messaging page
```
Comment (ryanluo) [inline src/main/webapp/js/user-page-loader.js:49]: nit: period :P

File: src/main/webapp/js/user-page-loader.js [hunk 2]
```diff
@@ -57,6 +58,19 @@ function setPageTitle() {
            const messageForm = document.getElementById('message-form');
            messageForm.action = '/messages?recipient=' + parameterUsername;
            messageForm.classList.remove('hidden');
+
+           /** Using 34 because @codestudents.com is 17 characters long
```
Comment (ryanluo) [inline src/main/webapp/js/user-page-loader.js:62]: So you're capping the username at 17? I don't quite get this logic.

File: src/main/webapp/js/user-page-loader.js [hunk 3]
```diff
@@ -57,6 +58,19 @@ function setPageTitle() {
...
+            * and there's at least 2 people in a direct message
+            */
+           if (parameterUsername.length < 34) {
+             const privateOption = document.getElementById('private-option');
```
Comment (ryanluo) [inline src/main/webapp/js/user-page-loader.js:66]: Is this a new Java thing? No type necessary?

Comment (ryanluo) [reply src/main/webapp/js/user-page-loader.js:66]: What happens if 'private-option' isn't in the doc?

File: target/codeu-starter-project-0.0.1-SNAPSHOT/js/user-page-loader.js [hunk 4]
```diff
@@ -46,6 +46,7 @@ function setPageTitle() {
```
Comment (ryanluo) [inline target/codeu-starter-project-0.0.1-SNAPSHOT/js/user-page-loader.js:46]: Do we need to track this file?

File: target/codeu-starter-project-0.0.1-SNAPSHOT/user-page.html [hunk 5]
```diff
@@ -41,19 +41,23 @@ <h1 id="page-title">User Page</h1>
       </div>
```
Comment (ryanluo) [inline target/codeu-starter-project-0.0.1-SNAPSHOT/user-page.html:41]: Do we need to track this file?

**Review by ryanluo — DISMISSED** [review]
Had some questions. Great work! Can we remove tracking of the files we don't care about?

File: src/main/webapp/js/user-page-loader.js (hunk 2 above)
Comment (kassidywebb) [reply src/main/webapp/js/user-page-loader.js:62]: Ideally I'd be able to check and see if it's a 2-user page... Right now, the direct messaging page link is 2 user's pages link pushed together. EG: ktwebb@codeustudents.comrluo@codeustudents.com

I'd rather figure out a more explanatory way to figure out it's 2 people, but for now, if there's 2 people, the user field in the link has to be greater than 34 characters. Basically, it's something I'll have to work on, but this is a temporary fix.

File: src/main/webapp/js/user-page-loader.js (hunk 3 above)
Comment (kassidywebb) [reply src/main/webapp/js/user-page-loader.js:66]: I basically copied it from previous code that was given to us (const messageForm, right above this). They didn't put a type, so I didn't either and I didn't receive any errors. Also, nothing would happen if it wasn't in the doc, it's like css files, you can have style specifications for a class, but if the class isn't in the file, nothing happens.

File: target/codeu-starter-project-0.0.1-SNAPSHOT/js/user-page-loader.js (hunk 4 above)
Comment (kassidywebb) [reply target/codeu-starter-project-0.0.1-SNAPSHOT/js/user-page-loader.js:46]: Nope :) Added a .gitignore.

File: target/codeu-starter-project-0.0.1-SNAPSHOT/user-page.html (hunk 5 above)
Comment (kassidywebb) [reply target/codeu-starter-project-0.0.1-SNAPSHOT/user-page.html:41]: Nope :) Added a .gitignore.

File: src/main/webapp/js/user-page-loader.js (hunk 1 above)
Comment (kassidywebb) [reply src/main/webapp/js/user-page-loader.js:49]: Deleted :)

**Comment by Ajide49** [comment]
Nice code! You did a great job documenting and spacing your implementations. Your explanation is also really good, I understand the changes you're making to implement the desired feature. I think Ryan hit pretty much all the little nitpicks about it.

**Comment by vado21** [comment]
I extremely recommend you not to merge this! I recommend you better if you want to do it privately, work on the direct messaging page. It is better!

**Pull request merged**